modules/fedimint-ln-client/src/pay.rs

```diff
@@ -247,7 +256,7 @@ impl LightningPayCreatedOutgoingLnContract {
 pub struct LightningPayFunded {
     payload: PayInvoicePayload,
     gateway: LightningGateway,
-    timelock: u32,
+    pub timelock: u32,
 }
```

Comment:
Since we are doing another iteration on this we should include the creation/funding time here. Both are close enough to each other normally, so I'd go with the funding time (avoids changing the created state).

That way the trigger future trying to contact the gateway stays inactive after the first n minutes of trying. Currently the code retries to reach the gateway for `TIMEOUT_DURATION` after each restart.

Reply:
Fixed. Added the `funding_time` to the funded state